buildout.d/versions.cfg

```diff
@@ -2,6 +2,7 @@
 setuptools = 8.3
 zc.buildout = 2.2.5
 zc.recipe.egg = 2.0.1
+plone.app.widgets = 1.8.0
```

Comment:
Yes, this is biting others---I think it's something to do with p.a.contenttypes & p.a.event.